src/main/java/com/crystalcraftmc/crystalstrike/BuildCrystal.java

```diff
@@ -0,0 +1,190 @@
+import java.util.List;
+
+import org.bukkit.Location;
+import org.bukkit.event.weather.LightningStrikeEvent;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+
+
+
+
+public class BuildCrystal {
+	
+	private Location baseLocation;
+	private final LightningStrikeEvent event;
+	
+	public BuildCrystal(final LightningStrikeEvent e){
+		baseLocation = e.getLightning().getLocation();
+		event = e;
+	}
+	
+	public void firstCrystal(){
+		
+		event.getLightning().getLocation().getWorld().getBlockAt(baseLocation.getBlockX(), baseLocation.getBlockY(), baseLocation.getBlockZ()).setType(Material.CHEST);//0,60,0 (original)
+        event.getLightning().getLocation().getWorld().getBlockAt(baseLocation.getBlockX() - 1, baseLocation.getBlockY(), baseLocation.getBlockZ()).setType(Material.PACKED_ICE);//-1,60,0
+        event.getLightning().getLocation().getWorld().getBlockAt(baseLocation.getBlockX() + 1, baseLocation.getBlockY(), baseLocation.getBlockZ()).setType(Material.PACKED_ICE);//1,60,0
+        event.getLightning().getLocation().getWorld().getBlockAt(baseLocation.getBlockX(), baseLocation.getBlockY(), baseLocation.getBlockZ() + 1).setType(Material.PACKED_ICE);//0,60,1
+        event.getLightning().getLocation().getWorld().getBlockAt(baseLocation.getBlockX(), baseLocation.getBlockY(), baseLocation.getBlockZ() - 1).setType(Material.PACKED_ICE);//0,60,-1
+        event.getLightning().getLocation().getWorld().getBlockAt(baseLocation.getBlockX() + 1, baseLocation.getBlockY(), baseLocation.getBlockZ() + 1).setType(Material.ICE);//1,60,1
+        event.getLightning().getLocation().getWorld().getBlockAt(baseLocation.getBlockX() - 1, baseLocation.getBlockY(), baseLocation.getBlockZ() + 1).setType(Material.ICE);//-1,60,1
+        event.getLightning().getLocation().getWorld().getBlockAt(baseLocation.getBlockX() - 1, baseLocation.getBlockY(), baseLocation.getBlockZ() - 1).setType(Material.ICE);//-1,60,-1
+        event.getLightning().getLocation().getWorld().getBlockAt(baseLocation.getBlockX() + 1, baseLocation.getBlockY(), baseLocation.getBlockZ() - 1).setType(Material.ICE);//1,60,-1
+        event.getLightning().getLocation().getWorld().getBlockAt(baseLocation.getBlockX(), baseLocation.getBlockY() + 1, baseLocation.getBlockZ()).setType(Material.OBSIDIAN);//0,61,0
+        event.getLightning().getLocation().getWorld().getBlockAt(baseLocation.getBlockX(), baseLocation.getBlockY() - 1, baseLocation.getBlockZ()).setType(Material.OBSIDIAN);//0,59,0
+        
+        //Adding items to the chest...how do we want to do this?
+        Block block = baseLocation.getBlock();
+        Chest chest = (Chest)block.getState();
+        Inventory inv = chest.getInventory();
+        
+        List<ItemStack> items = new ArrayList<ItemStack>();
+        items.add(new ItemStack(Material.DIAMOND_SWORD, 1));
+        items.add(new ItemStack(Material.DIAMOND_CHESTPLATE, 1));
+        items.add(new ItemStack(Material.ENDER_PEARL, 16));
```

Comment:
It would be cool to make all of these configurable! Maybe a better long-term task. Consider [opening an "issue"](https://github.com/CrystalCraftMC/CrystalStrike/issues/new) on this repo to use it as a feature tracker.